pkgs/applications/editors/neovim/utils.nix

```diff
@@ -193,11 +194,37 @@ let
     in
         lib.concatStringsSep ";" hostProviderLua;
 
+  grammarToPlugin = grammar:
+    let
+      name = lib.pipe grammar [
+        lib.getName
+
+        # added in buildGrammar
+        (lib.removeSuffix "-grammar")
+
+        # grammars from tree-sitter.builtGrammars
+        (lib.removePrefix "tree-sitter-")
+        (lib.replaceStrings [ "-" ] [ "_" ])
+      ];
+    in
+
+    runCommand "nvim-treesitter-grammar-${name}"
```

Comment:
maybe I should change the name to suffix it with "plugin". I also should call `toVimPlugin` on it